pulsar-flink-connector/src/test/java/org/apache/flink/streaming/connectors/pulsar/FlinkPulsarITest.java

```diff
@@ -173,7 +173,6 @@ public void testCaseSensitiveReaderConf() throws Exception {
         List<Integer> messages =
                 IntStream.range(0, 50).mapToObj(t -> Integer.valueOf(t)).collect(Collectors.toList());
 
-        sendTypedMessages(tp, SchemaType.INT32, messages, Optional.empty());
```

Comment:
Why do you want to delete here?

Reply:
This method default create topic, To test the automatic creation of a topic when it does not exist